packages/uhk-web/src/app/shared.module.ts

```diff
@@ -251,7 +252,8 @@ import { KeyActionDragAndDropService } from './services/key-action-drag-and-drop
         UhkDeviceUninitializedGuard,
         UhkDeviceLoadingGuard,
         UhkDeviceLoadedGuard,
-        UhkDeviceBootloaderNotActiveGuard
+        UhkDeviceBootloaderNotActiveGuard,
+        ThemeService
```

Comment:
The `ThemeService` import is not needed because you use the `{ providedIn: 'root' }` options